docker-app/qfieldcloud/core/models.py

```diff
@@ -1454,9 +1519,7 @@ def problems(self) -> list[dict]:
                                 "code": "layer_problem",
                                 "description": _(
                                     'Layer "{}" does not support the `primary key` attribute. The layer will be read-only on QField.'
-                                ).format(
-                                    layer_name,
-                                ),
+                                ).format(layer_name),
```

Comment:
```suggestion
                                ).format(
                                    layer_name,
                                ),
```
The newline was on purpose to keep it very clear where the variables are coming from. If you check the whole `problems` function is following this for maintainability.

Reply:
Changed!